pkg/repository/udmrepo/kopialib/lib_repo.go

```diff
@@ -600,6 +600,19 @@ func writeInitParameters(ctx context.Context, repoOption udmrepo.RepoOptions, lo
 			logger.Infof("Quick maintenance interval change from %v to %v", p.QuickCycle.Interval, overwriteQuickMaintainInterval)
 			p.QuickCycle.Interval = overwriteQuickMaintainInterval
 		}
+		fullMaintIntervalOption := udmrepo.FullMaintenanceIntervalOptions(repoOption.StorageOptions[udmrepo.StoreOptionKeyFullMaintenanceInterval])
+		if fullMaintIntervalOption == udmrepo.FastGC {
+			logger.Infof("Full maintenance interval change from %v to %v", p.FullCycle.Interval, udmrepo.FastGCInterval)
+			p.FullCycle.Interval = udmrepo.FastGCInterval
+		}
+		if fullMaintIntervalOption == udmrepo.EagerGC {
+			logger.Infof("Full maintenance interval change from %v to %v", p.FullCycle.Interval, udmrepo.EagerGCInterval)
+			p.FullCycle.Interval = udmrepo.EagerGCInterval
+		}
+		if fullMaintIntervalOption == udmrepo.NormalGC {
+			logger.Infof("Full maintenance interval change from %v to %v", p.FullCycle.Interval, udmrepo.NormalGCInterval)
+			p.FullCycle.Interval = udmrepo.NormalGCInterval
```

Comment:
@mpryc @Lyndon-Li Do you have any opinions as to if it should apply to already created repos? Or only from init only?

If we want to make it work for already created repos, will have to set maintenance options during repo open or during maintenance run.

Reply:
I don't think it is a good option to do this with every repo open/maintenance.
Alternatively, if users want to change the existing repo anyway, they can use kopia cli.

Reply:
Thanks

Reply:
We need to doc info for kopia cli debug
- User needs to know compatible kopia CLI version